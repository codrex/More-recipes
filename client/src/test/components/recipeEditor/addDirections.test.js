/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import toJson from 'enzyme-to-json';
import { PureAddDirections }
  from '../../../components/pages/RecipeEditor/AddDirections';


const props = {
  clearValidationError: jest.fn(),
  directions: true,
  externalError: [],
  handleSubmit: jest.fn(),
  initialize: jest.fn(),
  items: ['directions one', 'directions two'],
  name: 'directions',
  placeholder: 'add a direction',
  sendItemsToStore: jest.fn(),
};
describe('Add directions component', () => {
  test('expected to render add directions component', () => {
    const component = shallow(<PureAddDirections {...props} />);
    const tree = toJson(component);
    expect(component.find('AddItems').length).toBe(1);
    expect(tree).toMatchSnapshot();
  });
});

