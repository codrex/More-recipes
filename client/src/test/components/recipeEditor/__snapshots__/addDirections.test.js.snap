// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add directions component expected to render add directions component 1`] = `
<AddItems
  Component={null}
  clearValidationError={[Function]}
  directions={true}
  externalError={Array []}
  handleSubmit={[Function]}
  ingredients={false}
  initialize={[Function]}
  items={
    Array [
      "directions one",
      "directions two",
    ]
  }
  name="directions"
  placeholder="add a direction"
  sendItemsToStore={[Function]}
/>
`;
